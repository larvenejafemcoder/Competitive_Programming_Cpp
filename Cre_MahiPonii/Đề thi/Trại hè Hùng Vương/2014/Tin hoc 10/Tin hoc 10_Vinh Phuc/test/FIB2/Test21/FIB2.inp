5
32 893668
19 2260
20 8110
3 3
41 48609247
