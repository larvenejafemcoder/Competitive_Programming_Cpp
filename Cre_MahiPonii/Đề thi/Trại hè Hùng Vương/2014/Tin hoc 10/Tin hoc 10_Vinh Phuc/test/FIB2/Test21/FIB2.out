341351
863
3098
1
18567080
